.github/workflows/test.yaml

```diff
@@ -133,7 +220,7 @@ jobs:
           --timeout=90 \
           --ddtrace \
           --durations=5 \
-          ./integrations ./legacy ./trace_server ./trace ./tests
+          ./legacy
```

Comment:
This should be removed. Anything depending on the `client` fixture in legacy is misplaced. I just verified this by looking at the 3 cases

Reply:
ok, I found 4 cases and commented them out for now.

Reply:
I'll bring `test_weave_finish_unsets_client` over.

I don't think the Monitoring tests are valuable anymore.  We don't have that path, and the OpenAI and basic logging stuff has its own set of tests already.